src/utils.js

```diff
@@ -7,7 +7,9 @@ import fsExtra from 'fs-extra';
 import ApifyClient from 'apify-client';
 import psTree from '@apify/ps-tree';
 import requestPromise from 'request-promise-native';
-import cheerio from 'cheerio';
+import cheerio, { CheerioStatic } from 'cheerio'; // eslint-disable-line no-unused-vars
+import { ServerResponse } from 'http'; // eslint-disable-line no-unused-vars
+import { Response as PuppeteerResponse } from 'puppeteer'; // eslint-disable-line no-unused-vars
 import log from 'apify-shared/log';
```

Comment:
since `log` is coming from `apify-shared-js`, I think the docs should come from there, as of now, `log` has only `logJson` boolean property

Reply:
As far as I know, `apify-shared` is shared between the SDK and the "API backend" ("platform"/"product"). I think this is beyond the scope of this PR, but feel free to bring it up on the SDK channel or at the first suitable opportunity. Very similar opportunity comes with the `apify-client` module.

Reply:
Still working on this. There are 2 possible&easy temporary solutions involving just the `apify-js` package:
1. Map each exported entity from `apify-shared/log` to a JSdoc annotated entity in a new file `src/utils_log.js`. This means updated to the `apify-shared` package would need to be reflected in that file.
2. Manually write declarations file and _somehow_ include it into the generated declarations. This is preferred because one day `apify-shared` is likely to be typed too.

The problem is, `tsc` is difficult to persuade to use the `utils_log.d.ts` I wrote.

Reply:
Done. I added the declarations manually into `src/` and created a post-processing step fixing the `index.d.ts` generated by `tsc`.